sda/schemas/isolated/rotate-key.json

```diff
@@ -0,0 +1,29 @@
+{
+    "title": "JSON schema for SDA key rotation message interface",
+    "$id": "https://github.com/neicnordic/sensitive-data-archive/tree/master/sda/schemas/isolated/rotate-key.json",
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "required": [
+        "type",
+        "file_id"
+    ],
+    "additionalProperties": true,
+    "properties": {
+        "type": {
+            "$id": "#/properties/type",
+            "type": "string",
+            "title": "The message type",
+            "description": "The message type",
+            "const": "key_rotation"
+        },
+        "file_id": {
+            "$id": "#/properties/file_id",
+            "type": "string",
+            "title": "The unique file identifier",
+            "description": "The unique file identifier",
+            "examples": [
+                "420420cc43-e060-4583-a891-9f8170ee66c8"
+            ]
+        }
+    }
+}
```

Comment:
Since this file is identical to the other we can symlink them so we only have to edit this in one place.